pkg/api/product_cache.go

```diff
@@ -0,0 +1,108 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+const (
+	// Default is the standard real-time stats available to all services
+	Default = "default"
+	// OriginInspector is the product name used to determine access to Origin Inspector via the entitlement API
+	OriginInspector = "origin_inspector"
+)
```

Comment:
These identifiers resolve to `api.Default` and `api.OriginInspector` which are somewhat ambiguous.

```go
const (
	// ProductDefault represents the standard real-time stats available to all services.
	ProductDefault = "default"

	// ProductOriginInspector represents the origin inspector stats available via the
	// entitlement API.
	ProductOriginInspector = "origin_inspector"
)
```

Reply:
This much clearer, thank you.